features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/utils/DrawableUtils.kt

```diff
@@ -103,6 +108,19 @@ internal class DrawableUtils(
         fun onFailure()
     }
 
+    @VisibleForTesting
+    internal fun getBitmapSizeLimit(requestedSizeInBytes: Int?): Int {
```

Comment:
this is more like `resolveBitmapSizeLimit`

Reply:
This method decides between one of two constants (or returns the input if the input wasn't null) - is it not a getter? To me, 'resolve' implies that it might not return a result or that work needs to be done to reach the result.